script/vsts/platforms/templates/cache.yml

```
@@ -12,19 +12,28 @@ steps:
     displayName: Cache node_modules
     inputs:
       key: 'npm | "$(Agent.OS)" | "$(BUILD_ARCH)" | package.json, package-lock.json, script/vsts/platforms/${{ parameters.OS }}.yml'
```

I think we should probably replace `script/vsts/platforms/${{ parameters.OS }}.yml` with `script/vsts/platforms/templates/preparation.yml` for each of these three caches.
```suggestion
      key: 'npm | "$(Agent.OS)" | "$(BUILD_ARCH)" | package.json, package-lock.json, script/vsts/platforms/templates/preparation.yml'
```